app/index.html

```diff
@@ -5,22 +5,107 @@
     <link rel="stylesheet" media="screen" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.5/css/bootstrap.min.css">
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
     <title>Casa de Tintas NA </title>
-<style>
-
-</style>
-
 </head>
-
 <body>
-    <h1>Casa de tintas N.A.</h1>
-    <p>jdoifejorpje fdamkjçpoiqrjek fd,m mmdkjaoijeorijjfaujhpuhujejf</p>
+    <nav class="navbar navbar-inverse" style="border-radius:0px" id="caput">
+      <input type="text"  placeholder="Search" id="search" >
+        <ul class="nav nav-pills" id="register">
+            <li id="login" ><a href="#"><img src="images/login2.png" class="image">Login</a></li>
+            <li id="register"><a href="#"><img src="images/register.png" class="image">Register</a></li>
+        </ul>
+          <div id="figures" class="hide">
+           <a href="#"><img src="images/login2.png" class="image"></a>
+           <a href="#"><img src="images/register.png" class="image"></a>
+          </div>
+    </nav>
+
+    
+    <div id="carousel">
+        
+    <div class="header">
+        <div class="navbar">
+        <div class="container-fluid" id="secondNavBar">
+        <a class="navbar-brand" href="#">Casa de tintas N.A.</a>
+        <div class="navbar-right">
+            <a class="btn btn-navbar navbar-toggle" data-toggle="collapse" data-target=".navHeaderCollapse">
+                <span class="icon-bar"style="background-color:black"></span>
+                <span class="icon-bar"style="background-color:black"></span>
+                <span class="icon-bar"style="background-color:black"></span>
+            </a>
+            <div class="navbar-collapse collapse navHeaderCollapse">
+                <ul class="nav navbar-nav pull-right" id="colorful">  
+                    <li><a href="#" style="border-top-color: yellow" class="col-md-1 col-md-offset-3">HOME</a></li>
+                    <li><a href="#" style="border-top-color: orange" class="col-md-1">TYPOGRAPHY</a></li>
+                    <li><a href="#" style="border-top-color: pink" class="col-md-1">BLOG</a></li>
+                    <li><a href="#" style="border-top-color: violet" class="col-md-1">PAGES</a></li>
+                    <li><a href="#" style="border-top-color: purple" class="col-md-1">ELEMENTS</a></li>
+                    <li><a href="#" style="border-top-color: darkblue" class="col-md-1">ECOMMERSE</a></li>
+                    <li><a href="#" style="border-top-color: lightblue" class="col-md-1"> MENU</a></li>
+                </ul>   
+               </div>
+            </div>
+           </div>
+          </div>
+        
+        
+    </div>
+    <div id="carousel-example-generic" class="carousel slide" data-ride="carousel">
+        
+        <ul class="carousel-inner">
+            <li class="item active">
+                <h1 style="color:#01BBAC">Creating a unique look.</h1><br/><h1 style="color:white">Never been easier.</h1>
```

Comment:
you should avoid inline styles. Create classes that you will be able to reuse later